Sources/TuistCore/Graph/DependenciesGraph.swift

```diff
@@ -2,14 +2,14 @@ import ProjectDescription
 
 /// A directed acyclic graph (DAG) that Tuist uses to represent the dependency tree.
 public struct DependenciesGraph: Equatable, Codable {
-    /// A dictionary where the keys are the names of dependencies, and the values are the dependencies themselves.
-    public let externalDependencies: [String: [TargetDependency]]
+    /// A dictionary of Platforms to a dictionary where the keys are the names of dependencies, and the values are the dependencies themselves.
+    public let externalDependencies: [Platform: [String: [TargetDependency]]]
 
-    /// A dictionary where the keys are the folder of external projects, and the values are the projects themselves.
+    /// A dictionary where the keys are the supported platforms and the values are dictionaries where the keys are the names of dependencies, and the values are the dependencies themselves.
```

Comment:
This description shouldn't be changed as the property hasn't changed, isn't it?

Reply:
The property now also have the platform dictionary, so the property type has changed so I also changed the description to reflect that

Reply:
Sorry, I misread the line .. again. Will revert the change